src/core/store/interface.ts

```diff
@@ -29,7 +26,7 @@ export interface ReadStream {
  * PassThroughCallback: a callback intended for receiving a write stream to create a batch file
  */
 export interface PassThroughCallback {
-  (stream: ReadWriteStream): Promise<void>;
+  (stream: WriteStream): Promise<void>;
```

Comment:
undo to `(stream: ReadWriteStream): Promise<void>;`

Reply:
@enddynayn We currently only use the WriteStream part of this. Is this needed for the Read part? (I'll go check #66)